src/qt/managenamespage.h

```diff
@@ -0,0 +1,56 @@
+#ifndef MANAGENAMESPAGE_H
+#define MANAGENAMESPAGE_H
+
+#include <qt/platformstyle.h>
+
+#include <QWidget>
+
+class WalletModel;
+class NameTableModel;
+
+namespace Ui {
+    class ManageNamesPage;
+}
+
+QT_BEGIN_NAMESPACE
+class QTableView;
+class QItemSelection;
+class QSortFilterProxyModel;
+class QMenu;
+class QModelIndex;
+QT_END_NAMESPACE
+
+/** Page for managing names */
+class ManageNamesPage : public QWidget
+{
+    Q_OBJECT
+
+public:
+    explicit ManageNamesPage(const PlatformStyle *platformStyle, QWidget *parent = nullptr);
+    ~ManageNamesPage();
+
+    void setModel(WalletModel *walletModel);
+
+private:
+    const PlatformStyle *platformStyle;
+    Ui::ManageNamesPage *ui;
```

Comment:
I think we had a discussion about this previously, and it may just be how things are done with Qt (which I don't know anything about).  But in general, I think it would be better to use `std::unique_ptr` here instead of a raw pointer.

Reply:
I just ran `grep "Ui::" ./src/qt/*.h`, and it indicates that a raw pointer is consistent with upstream, so I prefer to keep it like this.

Reply:
Yes, I think most of the code does that, so it may be a general Qt pattern (or it may just be legacy that has not been corrected yet).  I'm definitely fine with following the existing Qt code.